plugins/resemble/src/models.ts

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2024 LiveKit, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Output format for Resemble TTS
+ */
+export type OutputFormat = 'wav' | 'mp3';
+
+export type Precision = 'PCM_16' | 'PCM_24' | 'PCM_32' | 'MULAW';
```

Comment:
have you tested that this works with mp3 and µ-law? we currently don't support them in ElevenLabs because we assume the input packet is always raw PCM